digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculator.scala

```diff
@@ -23,8 +23,11 @@ object ProjectDeltaCalculator {
     val projectLinks = ProjectDAO.getProjectLinks(projectId).groupBy(l => RoadPart(l.roadNumber,l.roadPartNumber))
     val currentAddresses = projectLinks.keySet.map(r => r -> RoadAddressDAO.fetchByRoadPart(r.roadNumber, r.roadPartNumber, true)).toMap
     val terminations = findTerminations(projectLinks, currentAddresses)
-
-    Delta(project.startDate, terminations.sortBy(t => (t.discontinuity.value, t.roadType.value)))
+    val newCreations = findNewCreations(projectLinks)
+    if (terminations.size + newCreations.size != currentAddresses.values.flatten.size)
```

Comment:
i think this is comparison between number of links and number of "templates" (roadnumber,roadpart) in the project